src/modules/users/components/Inbox/InboxItem/InboxItem.tsx

```diff
@@ -203,7 +198,7 @@ const InboxItem = ({
                     <Numeral
                       suffix={` ${token ? token.symbol : ''}`}
                       integerSeparator=""
-                      value={value}
+                      value={value || '-'}
```

Comment:
Why do we need to pass '-' and not just empty string? It seems that they both work but empty string is more of a convention in our code, unless there is a reason?

Reply:
The reason is, i checked how we was using it everywhere, and we was indeed putting '-' as fallback